core/src/ops/math/mod.rs

```diff
@@ -579,6 +579,11 @@ element_wise!(neg, Neg, [i8, i16, i32, i64, f16, f32, f64, TDim] => |_, xs| {
 };
 q: [i8, u8, i32] => |x: f32| -x);
 
+element_wise!(copy, Copy, [i8, i16, i32, i64, f16, f32, f64, TDim] => |_, _| {
+    Ok(())
+};
+q: [i8, u8, i32] => |x: f32| x);
```

Comment:
Not really sure what this `q` part does, for now just copied from the other ops

Reply:
The q section is for dealing with quantized datum types by converting to f32.